tests/v2/validation/charts/webhook.go

```diff
@@ -0,0 +1,57 @@
+package charts
+
+import (
+	"github.com/rancher/rancher/tests/framework/clients/rancher"
+	"github.com/rancher/rancher/tests/framework/extensions/kubeapi/webhook"
+)
+
+const (
+	localWebhookcount      = 14
+	downstreamWebhookCount = 7
```

Comment:
Just a couple of ideas here:

> Since these values won't always be 14 and 7.

1. Should we make these counts passable in through the config, so it'd be on whoever is running the test to know these counts?
2. Should we make the check just to ensure that it's greater than zero. As zero is the complete fail state, but wouldn't catch partial failures
3. Or just make sure someone comes to update these numbers with PRs once these numbers change?

Reply:
I'm curious about these numbers as well, but I don't think the value should be configurable though Andrew. We should have a set way of validating it.

Reply:
Ok, I'll update it to mae sure we have the count as greater than zero and log the count of webhooks and list of webhooks that we have in our rancher server. That way we are not asserting the count and list and having to re work each release when there's change and also not limiting ourselves to not having the info about the webhooks that we have on rancher?